src/Plugins/SimplnxCore/test/VerifyTriangleWindingTest.cpp

```diff
@@ -0,0 +1,200 @@
+
+#include "simplnx/DataStructure/Geometry/TriangleGeom.hpp"
+#include "simplnx/Parameters/DataGroupSelectionParameter.hpp"
+#include "simplnx/UnitTest/UnitTestCommon.hpp"
+#include <catch2/catch.hpp>
+
+#include "SimplnxCore/Filters/LabelTriangleGeometryFilter.hpp"
+#include "SimplnxCore/Filters/ReadStlFileFilter.hpp"
+#include "SimplnxCore/Filters/VerifyTriangleWindingFilter.hpp"
+#include "SimplnxCore/SimplnxCore_test_dirs.hpp"
```

Comment:
The include block should be sorted from the most specific (current plugin) to the least. I think for the tests, the "catch2" must be first.

#include <catch2/catch.hpp>

#include "SimplnxCore/Filters/LabelTriangleGeometryFilter.hpp"
#include "SimplnxCore/Filters/ReadStlFileFilter.hpp"
#include "SimplnxCore/Filters/VerifyTriangleWindingFilter.hpp"
#include "SimplnxCore/SimplnxCore_test_dirs.hpp"

#include "simplnx/DataStructure/Geometry/TriangleGeom.hpp"
#include "simplnx/Parameters/DataGroupSelectionParameter.hpp"
#include "simplnx/UnitTest/UnitTestCommon.hpp"